### PR TITLE
[4.0] Updated package generation according to the new specs structure

### PR DESCRIPTION
**Pull request by rauldpm**
|Related issue|
|---|
|https://github.com/wazuh/wazuh-packages/issues/765|

## Description

This PR updates the references when cloning or downloading the wazuh-packages repository with respect to the 4.0 branch.

It also updates the help parameters as much as possible and adds the parameters necessary for a basic use.

## Checks
- [x] It compiles without warnings.
- [x] Spelling and grammar. 
- [x] Used impersonal speech. 
- [ ] Used uppercase only on nouns. (not used)
- [ ] Updated the `redirect.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).

### CODE REVIEW

File: source/development/packaging/generate-aix-package.rst [hunk 1]
```diff
@@ -18,7 +18,7 @@ Download our wazuh-packages repository from GitHub and go to the aix directory.
 
 .. code-block:: console
 
- $ curl -L https://github.com/wazuh/wazuh-packages/tarball/master | tar zx
+ $ curl -L https://github.com/wazuh/wazuh-packages/tarball/4.0 | tar zx
```
Comment (alberpilot) [inline source/development/packaging/generate-aix-package.rst:21]: Add a variable `WAZUH_PACKAGES_BRANCH` to `conf.py` and replace all harcoded 4.0 values

**Review by alberpilot — CHANGES_REQUESTED** [review]
Please review the requested changes

File: source/development/packaging/generate-aix-package.rst (hunk 1 above)
Comment (rauldpm) [reply source/development/packaging/generate-aix-package.rst]: Fixed in https://github.com/wazuh/wazuh-documentation/pull/3864/commits/27a954b305156beaa2b68798932d0dc9cc7ea3ec

**Review by alberpilot — APPROVED** [review]

**Review by davidfmiranda — APPROVED** [review]
LGTM

**Pull request merged**